zookeeper-server/src/test/java/org/apache/zookeeper/audit/Log4jAuditLoggerTest.java

```diff
@@ -48,33 +37,37 @@
 import org.apache.zookeeper.server.quorum.QuorumPeerTestBase;
 import org.apache.zookeeper.test.ClientBase;
 import org.apache.zookeeper.test.ClientBase.CountdownWatcher;
+import org.apache.zookeeper.test.LoggerTestTool;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.LineNumberReader;
+import java.io.StringReader;
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+import java.util.ArrayList;
+import java.util.List;
 
 public class Log4jAuditLoggerTest extends QuorumPeerTestBase {
```

Comment:
Shall we rename this class to remove mention of Log4J?

Reply:
I should rename the class as well, not just the test.
Actually it should be `Slf4jAuditLogger` to be precise.